src/pal/src/shmemory/shmemory.cpp

```diff
@@ -455,14 +455,14 @@ BOOL SHMInitialize(void)
     init_waste();
     
 #ifndef CORECLR
```

Comment:
I'll be removing the ifndef blocks in another PR, don't want to club it with this change. And the changes here were made by @JeremyKuhne. which are not needed, as we don't support MAX_LONGPATH outside of coreclr. Leaving this as is, and cleaning up later.